.github/workflows/create-issue.yml

```diff
@@ -45,3 +45,7 @@ jobs:
           JIRA_API_TOKEN: ${{ secrets.JIRA_API_TOKEN }}
       - name: Add label
         run: jira labels add ${{ needs.create-issue.outputs.issueId }} 'switchboard'
+        env:
+          JIRA_BASE_URL: ${{ secrets.JIRA_BASE_URL }}
+          JIRA_USER_EMAIL: ${{ secrets.JIRA_USER_EMAIL }}
+          JIRA_API_TOKEN: ${{ secrets.JIRA_API_TOKEN }}
```

Comment:
I think these can be added in a shared way instead of being duplicated. This is how it was in one of the examples that you shared previously with me, as I recall